moveit_core/planning_scene/test/test_planning_scene.cpp

```diff
@@ -590,6 +590,61 @@ TEST(PlanningScene, RobotStateDiffBug)
   }
 }
 
+TEST(PlanningScene, ACMBug)
```

Comment:
optional: rather than calling the test `ACMBug`, how about something more descriptive like `UpdateACMAfterObjectRemoval` or so?

Reply:
I updated the name. Thanks!